napari/components/dims.py

```diff
@@ -252,6 +252,28 @@ def set_display(self, axis: int, display: bool):
             self._display[axis] = display
             self.events.display(axis=axis)
 
+    def swap_display(self, axis_a: int, axis_b: int):
+        """Swaps the display properties of two axes.
+
+        Parameters
+        ----------
+        axis_a : int
+            Dimension index
+        axis_b : int
+            Dimension index.
+        """
+        if axis_a < 0:
+            axis_a += self.ndim
+        if axis_b < 0:
+            axis_b += self.ndim
+
```

Comment:
i feel like we should be better about input checking in general (i.e. negatives, or longer than the list, etc. I could just take a modulus and it will work every time - `axis_b = axis_b % self.ndim`?

Reply:
fixed with `%`

Reply:
it would work every time but it doesn't make sense – if someone provides an index of `-5` when there are 2 axes that should raise an exception. you can see with lists, numpy, etc. it does not "wrap around" more than once and shouldn't even be thought about as wrapping around really, it's more like the index from the last element instead of the first